### ISSUE TITLE
Trouble connecting to localhost remotely

### ISSUE THREAD

**Issue by WestDesigns** [Varying-Vagrant-Vagrants/VVV#1008]
Having a unique issue here. Installed `vvv`, `variable vvv`, `vagrant hostsupdater`, and @leogopal's dashboard. I have a 2TB drive that is shared to my network with `vvv` installed on it. I am constantly running the box, and have created a `http://test.dev` site that I can see locally. Others on my network cannot, even though they can edit my files on the drive. They simply can't visit that domain to see the site. 

Obviously this is a localhost issue, of course other computers on the network (or remote) can't access `http://test.dev` and view the site. This is where I need some assistance. 

Anyone can use my IP:8080 to access the dashboard just fine. They can even access phpmyadmin and all the dev tools associated with it. But they cannot view any website in the dashboard except for me (`http://localhost.wordpress.dev, http://test.dev, etc). 

How can I allow them to view these sites? What am I missing here, a `vagrant dns` plugin? Is there any setting I've missed in the `Vagrantfile`? 

_tl:dr not in IT, need advice on remote connecting and viewing of vvv directories for development. I want to be able to view `http://test.dev:8080` from home but I am unable to._

If any formatting is off I apologize, new to git.

**Comment by JordanDevelop** [comment]
Make sure you make the necessary changes in the other computers host files. If you're on OS X, it's located at /etc/hosts. If you're on Windows it's C:\Windows\System32\drivers\etc\hosts

Find the private IP address of the virtual machine running vagrant by running ifconfig and looking at the "inet addr" under eth0. On the other machine (not your computer or the virtual machine) put inside the host file this but replacing 192.168.0.1 with your virtual machines IP address:

192.168.0.1 test.dev

That should fix the issue. If you want to make your vagrant machine public, as in people who aren't connected to your network can see your server, you should probably learn a little bit more before doing that. Opening ports to world isn't something you should play around with.

**Comment by WestDesigns** [comment]
Getting an `ERR_ICANN_NAME_COLLISION`

**Comment by JordanDevelop** [comment]
What pseudo domain are you using?

**Comment by JordanDevelop** [comment]
Put this in your Vagrantfile and reload:

config.vm.network "forwarded_port", guest: 80, host: 8080

**Comment by WestDesigns** [comment]
I'm not sure I am familiar with pseudo domains for VVV. 

Sure thing, I will add that in now and report.

**Comment by JordanDevelop** [comment]
For example, test.dev is considered a pseudo domain. A domain that doesn't technically exist.

**Comment by WestDesigns** [comment]
My apologies. For this example I'm trying to run project.dev as the pseudo, with my dashboard being vvv.dev.

**Comment by JordanDevelop** [comment]
Ok. So in the host file I mentioned above you're going to put

YOUR_VM_PRIVATE_IP project.dev
YOUR_VM_PRIVATE_IP vvv.dev

My virtual machines private IP is 192.168.0.27, so I put this in the host files.

192.168.0.27 project.dev
192.168.0.27 vvv.dev

**Comment by JordanDevelop** [comment]
Do you know what your Virtual Machines private IP is?

**Comment by agrahamwd** [comment]
All of this makes sense, still trying to locate the virtual machines private IP. I've been using my computers IPv4 address which could explain why it's not working.

**Comment by agrahamwd** [comment]
Still cannot find my virtual machines private IP, it's not showing up in ipconfig. Is there anywhere else I can definitely find it?

**Comment by agrahamwd** [comment]
@JordanDevelop this definitely was the answer! I am able to view these sites with my colleagues. Thanks so much for the help!
